ClientCore/ClientConfiguration.cs

```diff
@@ -184,6 +184,8 @@ public void RefreshSettings()
 
         public string LocalGame => clientDefinitionsIni.GetStringValue(SETTINGS, "LocalGame", "DTA");
 
+        public int SteamGameId => clientDefinitionsIni.GetIntValue(SETTINGS, "SteamGameId", 2229850);
```

Comment:
I don't think this should be configured to the default value of YR one explicitly. you should check if it's a YR or Ares config via ifdef and use that as default, elif TS - use Steam TS one, else assign a missing ID value (idk if 0 is correct for Steam?) and check before even initializing.

Reply:
![image](https://github.com/user-attachments/assets/9e7c65e7-c148-4529-a841-ef813b905c7f)
![image](https://github.com/user-attachments/assets/24f1b90a-4f1f-49b1-a5c9-0a9f7afc40a5)
What about this instead

Reply:
Why? It's not hard to change the default value for something sane based on the build config, and I don't think having am ability for people change steam app ID is something that should be exposed a lot. Besides, there are only two such games that are on Steam and are supported by the client - TS and YR.

Reply:
> What about this instead

Why Steam Game Identityficator can be set from `ClientDefinitions.ini`? Is there possible situation when modder publish its mod in the Steam as the standalone game? If not -- better to hardcode YR and TS Steam IDs as Kerbiter said.

In addition to the above, i believe a button to enable/disable feature (like DRP) is needed.